tests/integration/test_structured_logging_json/test.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+import logging
+import json
+from xml.etree import ElementTree
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node", stay_alive=True)
```

Comment:
It seems like because there is no config in which we specify `<formatting>` this test is skipped every time and does not check anything. So we need to add some `config/config_json.xml` file with content similar to [this](https://github.com/ClickHouse/ClickHouse/blob/master/tests/integration/test_logs_level/configs/config_information.xml) file (with `<level>trace</level>` and specified `<formatting>` setting) and add this config to the cluster.

Also it does not seem like we need stay_alive here
```suggestion
node = cluster.add_instance("node", main_configs=["configs/config_json.xml"])
```

Reply:
code updated